test/cpp/cpp_test_util.cpp

```diff
@@ -278,6 +285,15 @@ std::vector<at::Tensor> ExecuteAndFetch(absl::Span<const ir::Value> roots,
   return Fetch(results);
 }
 
+void AssertBackward(const torch::Tensor& xla_output,
+                    const torch::Tensor& xla_input,
```

Comment:
Can we generalize this to take `vector<torch::Tensor>`? For some op we take more than one input.

Reply:
done!